match/spec/utils_spec.rb

```diff
@@ -6,7 +6,7 @@
 
     describe 'import' do
       it 'finds a normal keychain name relative to ~/Library/Keychains' do
-        expected_command = "security import item.path -k '#{Dir.home}/Library/Keychains/login.keychain' -P #{''.shellescape} -T /usr/bin/codesign -T /usr/bin/security 1> /dev/null"
+        expected_command = "security import item.path -k #{Dir.home}/Library/Keychains/login.keychain -P #{''.shellescape} -T /usr/bin/codesign -T /usr/bin/security 1> /dev/null"
```

Comment:
Isn't this missing a `.shellescaoe` for the `-k` value here? (relevant e.g. when `Dir.home` has a space)

Reply:
@janpio, Yes, I noticed other cases where the `spec` files needed to be updated with `.shellescape` after I pushed my initial update. 

I have not been unable to get back to them, but I will address these today.

Reply:
I pushed an update to these.